File: react/ap-store-pos-api/src/main/java/com/apstore/api/pos/apstoreposapi/vo/OrderDetailsVo.java
package com.apstore.api.pos.apstoreposapi.vo;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@ToString
@Setter
public class OrderDetailsVo {
    private int quantity;

    private String itemName;

    private int category;

    private double cost;
    
}

File: react/ui-pos-app/src/model/orderdetailsvo.js
export default class OrderDetailsVo {
    constructor(_id,_quantity,_itemname,_category,_cost){
        this.id = _id;
        this.quantity = _quantity;
        this.itemName = _itemname;
        this.category = _category;
        this.cost = _cost;
    }
}